README.md

```diff
@@ -148,6 +152,14 @@ Run it
 $ kn event -h
 ```
 
+### Install using Homebrew
```

Comment:
I'd probably merge with "Install to work with `kn` CLI" and have something like "Install to work with `kn` CLI from Homebrew". And pointing to kn homebrew instructions first, or just adding the install cmd in a code block.

https://knative.dev/docs/client/install-kn/#install-the-knative-cli

Reply:
The reason why I added it as a separate heading on the same level as "Install using a binary" was, that you don't need to download the binary manually when using homebrew :shrug: